packages/logging/src/log-escalator.test.ts

```diff
@@ -0,0 +1,310 @@
+import pino from 'pino'
+import { logEscalator } from './log-escalator'
+
+it('does not write logs until done is called', () => {
```

Comment:
I'm a bit worried about this behaviour. If an error occurs its possible that a bug in the client code will result in done() never being called. In that situation you suddenly have no logs at all. Is it possible to avoid the need for the done() function? It would feel safer it it was 'once an error is logged you are getting logs no matter what'.

Reply:
Once done() is called once, the logs will be written immediately. Also done will automatically be called after 10 seconds.

There is a chance if the process crashes during that time we will lose logs. Very few things should crash a node process, the few times that has happened we haven't got logs because it just crashed out hard.